src/ng-xform/select-field/select-field.component.ts

```diff
@@ -53,6 +53,10 @@ export class SelectFieldComponent extends BaseDynamicFieldComponent<SelectField>
     this.select.registerOnTouched(this._onTouched);
     this.viewModel.subscribe((value: any) => {
       if (this.field.searchByValueAttributeHandler) {
+        if (value === undefined || value === null) {
```

Comment:
How could value be undefined?

Reply:
If initialize the FormControl with undefined value